test/Ocelot.UnitTests/Polly/PollyQoSResiliencePipelineProviderTests.cs

```diff
@@ -164,7 +164,7 @@ public async Task Should_throw_but_after_delay_should_allow_one_more_internal_se
         await Assert.ThrowsAsync<BrokenCircuitException>(async () =>
             await resiliencePipeline.ExecuteAsync((_) => ValueTask.FromResult(response)));
 
-        await Task.Delay(6000);
+        await Task.Delay(8000);
```

Comment:
@ggnaegi Gui, did you experiment with QoS tests? 😉  
Just ask me to re-run build in the chat! 🆗 ❔